drivers/input/pinnacle/Kconfig

```diff
@@ -0,0 +1,47 @@
+# Cirque Pinnacle 1CA027 touch controller based device configuration options
+#
+# Copyright (c) 2024 Ilia Kharin
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig INPUT_PINNACLE
+	bool "Cirque Pinnacle 1CA027 Touch Controller Based Device"
+	default y
+	depends on DT_HAS_CIRQUE_PINNACLE_ENABLED
+	select SPI
+	help
+	  Enable driver for 1CA027 SPI-based trackpad devices, for example
+	  GlidePoint modules.
+
+if INPUT_PINNACLE
+
+choice INPUT_PINNACLE_TRIGGER_MODE
+	prompt "Trigger mode"
+	default INPUT_PINNACLE_TRIGGER_OWN_THREAD
+	help
+	  Specify the type of triggering to be used by the driver.
+
+config INPUT_PINNACLE_TRIGGER_GLOBAL_THREAD
+	bool "Use global thread"
+	depends on GPIO
+
+config INPUT_PINNACLE_TRIGGER_OWN_THREAD
+	bool "Use own thread"
+	depends on GPIO
+
+endchoice
+
+config INPUT_PINNACLE_THREAD_PRIORITY
+	int "Thread priority"
+	depends on INPUT_PINNACLE_TRIGGER_OWN_THREAD
+	default 10
+	help
+	  Priority of thread used by the driver to handle interrupts.
+
+config INPUT_PINNACLE_THREAD_STACK_SIZE
+	int "Thread stack size"
+	depends on INPUT_PINNACLE_TRIGGER_OWN_THREAD
+	default 512
+	help
+	  Stack size of thread used by the driver to handle interrupts.
+
+endif # INPUT_PINNACLE
```

Comment:
This looks like sensor boilerplate leftover, we don't need any of that, all other input driver happily do their thing with just the workqueue, just drop this all stuff and keep workqueue mode, should simplify few things.

Reply:
@fabiobaltieri, sure, I'll update this part.

Reply:
Removed these parameters